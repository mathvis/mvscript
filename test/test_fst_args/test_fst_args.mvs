func a(b: int, c: string) {}
