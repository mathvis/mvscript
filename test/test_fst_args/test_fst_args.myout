(Decl (FunctionDef (FunctionIdentifier "a") [(VarIdentifier "b",IntT),(VarIdentifier "c",StringT)] VoidT (Just (Block FunctionBlock []))),ParserState {config = Configuration {debug = True, collapseOperations = True, collapseControlFlow = True, colors = True}, st = fromList [], fst = fromList [("a",FunctionData {returnType = VoidT, arguments = [("b",IntT),("c",StringT)], hasBody = True})]})
