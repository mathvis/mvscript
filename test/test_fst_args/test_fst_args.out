(Decl (FunctionDef (FunctionIdentifier "a") [(VarIdentifier "b",IntT),(VarIdentifier "c",StringT)] VoidT (Just (Block (FunctionBlock VoidT) []))),ParserState {config = Configuration {debug = True, collapseOperations = True, collapseControlFlow = True, colors = True}, st = fromList [], fst = fromList [("a",FunctionData {returnType = VoidT, arguments = [("b",IntT),("c",StringT)], hasBody = True})], context = Nothing, unresolvedFunctionCalls = []})
